spec/models/service_spec.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe Service, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  context 'database associations' do
+    it { is_expected.to have_and_belong_to_many(:events) }
+  end
+
+  context 'model attributes' do
+    it { is_expected.to have_db_column(:name).of_type(:string)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.